gcovr/templates/style.details.css

```diff
@@ -82,6 +82,12 @@ pre
     margin-bottom: 0;
 }
 
+.listOfFunctions > ul > li > a
+{
+    text-decoration: none;
+    color: inherit;
```

Comment:
When removing the underline and characteristic blue text, the links need some other affordance to show that they are interactive elements. For example, the following adds a background colour on hover.

```css
.listOfFunctions a {
    text-decoration: none;
    color: inherit;
    transition: 0.3s;
    padding: 0 4px;
}

.listOfFunctions a:hover, .listOfFunctions a:focus {
    background: lightsteelblue;
}
```

Reply:
I was also thinking about this. A background is even better than the underline. Should we add a icon via CSS after the link?

Reply:
what kind of icon would you suggest? Perhaps turning that list into a table that also shows the number of calls, and covered/uncovered status through the background color?

Reply:
Something like this:
<img width="406" alt="Bildschirmfoto 2021-09-19 um 12 11 40" src="https://user-images.githubusercontent.com/40258682/133923713-e7b6cb43-14a1-4626-b82e-ecbadf71286f.png">

Or a unicode character like her:
<img width="561" alt="Bildschirmfoto 2021-09-19 um 12 13 09" src="https://user-images.githubusercontent.com/40258682/133923740-e9a356fa-d6e7-4377-9fd9-8ff438204e7f.png">

Reply:
🤔 any affordance would do, as long as it's clear that the function name is clickable. It makes sense to remove `text-decoration: underline` since it tends to conflict with underscores in function names, but one or more hints like blue text color, interactive background color or outline, or symbols are necessary. Of the suggested symbols I only dislike the “box with arrow pointing outward” since it suggests an external site. The “→” would optically clash with the bullet point from the list, but that can be fixed with a “list-style: none”.

Reply:
I added a table with function level information. The rows have the same behavior as the toplevel filelist (alternating row colors and chanhing color on hover). What do you think about this?

<img width="842" alt="Bildschirmfoto 2021-09-19 um 23 11 37" src="https://user-images.githubusercontent.com/40258682/133943248-d2f2d6f4-3fb6-414a-89bb-caf6b6117415.png">

Reply:
Looks fine! There are some details I might change in a future PR, but I don't want to “bikeshed” the design so much that we never get around to releasing this improvement :)

It strikes me that this table would benefit greatly from linking call locations (perhaps as part of a separate PR) but I'm not sure how this can be done correctly as we can't know whether a function has internal linkage – there might be different functions with the same name in different compilation units. Call locations aren't yet represented in our coverage data model, but my new parser would make it easy to get this info.

Reply:
Than  I'll update the test data.